src/main/java/com/breskul/bring/annotations/Configuration.java

```diff
@@ -0,0 +1,18 @@
+package com.breskul.bring.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Indicates that a class declares one or more {@link Bean @Bean} methods and
+ * using by Bring for generating and getting bean instances.
+ *
+ */
+
+@Component
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Configuration {
```

Comment:
`Configuration` should have a `value` method to store the name of the bean.

Reply:
Done.